runtime/gc_glue_java/ConcurrentMarkingDelegate.cpp

```diff
@@ -431,6 +431,10 @@ MM_ConcurrentMarkingDelegate::concurrentClassMark(MM_EnvironmentBase *env, bool
 					}
 				}
 
+				if (classLoader == _javaVM->systemClassLoader) {
+					_markingScheme->markObject(env, (j9object_t)_javaVM->unamedModuleForSystemLoader->moduleObject);
```

Comment:
Do we need explicit cast to `(j9object_t)`? I think `J9Module-> moduleObject` has this type already